item.rb

```diff
@@ -0,0 +1,32 @@
+require 'date'
+
+class Item
+  attr_accessor :publish_date, :genre, :label, :author
+  attr_reader :id, :archived
+
+  def initialize(publish_date, archived, id = nil)
+    @id = id || rand(1..1000)
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def can_be_archived?
+    Date.today.year - Date.parse(@publish_date).year > 10
+  end
+
+  def move_to_archive
+    @archived = true if can_be_archived?
+  end
+
```

Comment:
- [ ] According to the project requirements there is a relationship between the Item and the author, which means they should have a method that makes a connection and is present in the Item class. I would like to recommend creating a method for that. Please take a look at the example below ⬇️  and make the change in order to make your project works as expected 👍 


```ruby
    def author=(author)
      @author = author
      author.items.push(self) unless author.items.include?(self)
    end
```

Reply:
Changes implemented

Reply:
Changes implemented